gradle.properties

```diff
@@ -0,0 +1,8 @@
+#Gradle properties
+quarkusPluginId=io.quarkus
+quarkusPluginVersion=3.7.3
+quarkusPlatformGroupId=io.quarkus.platform
+quarkusPlatformArtifactId=quarkus-bom
+quarkusPlatformVersion=3.7.3
+jaxrsFunctionalTestBuilderVersion=1.0.6
+okhttpVersion=3.12.13
```

Comment:
Should we use Quarkus 3.8 or 3.8.1? Those are the latest versions. Please check :)